### DISCUSSION TITLE
Is there a new release planned in the near future?

### DISCUSSION THREAD

**Discussion by Andreas-Schniertshauer** [nlohmann/json#2591] [Q&A] [answered]
Is there a new release planned where e.g. the actual thirdparty doctest 2.4.4 is added, I ask, because doctest 2.3.7 delivered with json-3.9.1.zip does not compile with /std:c++latest enabled, because doctest uses uncaught_exception that is deprecated in C++17 and removed in C++20.

**Comment by nlohmann** [comment, marked as answer]
There is no release data for 3.10.0 yet, and I am waiting to have some of the PRs merged before the release. I hope to have the release done before February.

**Reply by abrownsword** [reply]
It is now August...?

**Reply by nlohmann** [reply]
Yes...

**Reply by nlohmann** [reply]
https://github.com/sponsors/nlohmann ;-)

**Reply by nlohmann** [reply]
Done: https://github.com/nlohmann/json/releases/tag/v3.10.0
